src/bindings/js/node/src/infer_request.cpp

```diff
@@ -138,15 +138,33 @@ Napi::Value InferRequestWrap::get_output_tensor(const Napi::CallbackInfo& info)
     return TensorWrap::wrap(info.Env(), tensor);
 }
 
+std::map<std::string, ov::Tensor> get_js_infer_result(ov::InferRequest* infer_request) {
+    auto model_outputs = infer_request->get_compiled_model().outputs();
+    std::map<std::string, ov::Tensor> outputs;
```

Comment:
Here is better to use model output ID (number) to show all outputs.

In case model has 3 outputs but output 2,3 have same tensor (same names) then this map shows only two outputs but model has got three.